.tool-versions

```diff
@@ -0,0 +1,4 @@
+redis 6.2.7
+ruby 3.2.0
+nodejs 18.14.0
+yarn 1.22.19
```

Comment:
We need this top-level definition since these tools are required to build the repo modules with `turbo` (though not sure if Ruby is actually needed at the top-level)